x-pack/plugins/apm/public/components/app/service_overview/index.tsx

```diff
@@ -31,216 +34,225 @@ const TableLinkFlexItem = styled(EuiFlexItem)`
 `;
 
 interface ServiceOverviewProps {
+  agentName?: string;
   serviceName: string;
 }
 
-export function ServiceOverview({ serviceName }: ServiceOverviewProps) {
+export function ServiceOverview({
+  agentName,
+  serviceName,
+}: ServiceOverviewProps) {
   useTrackPageview({ app: 'apm', path: 'service_overview' });
   useTrackPageview({ app: 'apm', path: 'service_overview', delay: 15000 });
 
   return (
-    <EuiFlexGroup direction="column" gutterSize="s">
-      <EuiFlexItem>
-        <EuiFlexGroup
-          gutterSize="xs"
-          style={{ marginTop: 16, marginBottom: 8 }}
-        >
-          <EuiFlexItem grow={2}>
-            <EuiPanel>Search bar</EuiPanel>
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiPanel>Comparison picker</EuiPanel>
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiPanel>Date picker</EuiPanel>
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      </EuiFlexItem>
-      <LatencyChartRow>
-        <EuiPanel>
-          <EuiTitle size="xs">
-            <h2>
-              {i18n.translate('xpack.apm.serviceOverview.latencyChartTitle', {
-                defaultMessage: 'Latency',
-              })}
-            </h2>
-          </EuiTitle>
-        </EuiPanel>
-      </LatencyChartRow>
-      <Row>
-        <EuiFlexGroup gutterSize="s">
-          <EuiFlexItem grow={4}>
-            <EuiPanel>
-              <EuiTitle size="xs">
-                <h2>
-                  {i18n.translate(
-                    'xpack.apm.serviceOverview.trafficChartTitle',
-                    {
-                      defaultMessage: 'Traffic',
-                    }
-                  )}
-                </h2>
-              </EuiTitle>
-            </EuiPanel>
-          </EuiFlexItem>
-          <EuiFlexItem grow={6}>
-            <EuiPanel>
-              <EuiFlexGroup justifyContent="spaceBetween">
-                <EuiFlexItem>
-                  <EuiTitle size="xs">
-                    <h2>
-                      {i18n.translate(
-                        'xpack.apm.serviceOverview.transactionsTableTitle',
-                        {
-                          defaultMessage: 'Transactions',
-                        }
-                      )}
-                    </h2>
-                  </EuiTitle>
-                </EuiFlexItem>
-                <TableLinkFlexItem>
-                  <TransactionOverviewLink serviceName={serviceName}>
+    <ChartsSyncContextProvider>
+      <EuiFlexGroup direction="column" gutterSize="s">
+        <EuiFlexItem>
+          <EuiFlexGroup
+            gutterSize="xs"
+            style={{ marginTop: 16, marginBottom: 8 }}
+          >
+            <EuiFlexItem grow={2}>
+              <EuiPanel>Search bar</EuiPanel>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiPanel>Comparison picker</EuiPanel>
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiPanel>Date picker</EuiPanel>
+            </EuiFlexItem>
```

Comment:
I don't know if you are planning to handle it later, but maybe you should add translations here.

Reply:
This is just a placeholder until we do #81723.